app/views/incomeTaxHistory/IncomeTaxHistoryView.scala.html

```diff
@@ -104,7 +105,13 @@ <h2 class="govuk-heading-m govuk-!-margin-top-0" >@TaxPeriodLabelService.taxPeri
             <ul class="govuk-list govuk-!-font-size-16 .ul-incomeTaxHistory">
                 <li class ="li-incomeTaxHistory"><span class="ul-span-incomeTaxHistory">@Messages("tai.incomeTax.history.employerReference")</span> @summary.ern	</li>
                 <li class ="li-incomeTaxHistory"><span class="ul-span-incomeTaxHistory">@Messages("tai.incomeTax.history.startDate")</span> @TaxYearRangeUtil.formatDate(summary.startDate)	</li>
-                <li class ="li-incomeTaxHistory"><span class="ul-span-incomeTaxHistory">@Messages("tai.incomeTax.history.endDate")</span> @TaxYearRangeUtil.formatDate(summary.endDate)	</li>
+                <li class ="li-incomeTaxHistory"><span id="end-date" class="ul-span-incomeTaxHistory">@Messages("tai.incomeTax.history.endDate")</span>
+                    @{if(summary.endDate.isEqual(LocalDate.now))
```

Comment:
If you look at the controller you'll see the following: 
```employment.endDate.getOrElse(LocalDate.now()),```

Just make `IncomeTaxHistoryViewModel.endDate` optional (call it maybeEndDate).
And then ```@if(summary.maybeEnddate.isDefined)```

Reply:
👍 Incorporated